### ISSUE TITLE
xcode needed for FirmwareSender Compilation

### ISSUE THREAD

**Issue by kaosbeat** [pingdynasty/FirmwareSender#2]
Hi, 

Trying to compile for OSX, but (very) reluctant to download and install Xcode. Is there a simple way for you to add a generic make file? That would be very much appreciated. I tried compiling the linux build, but of course that did not work (alsa)

**Comment by mcclure** [comment]
I don't think you do need xcode. I think you just need to install the CommandLineTools package and then run `xcodebuild -configuration Release` in the Builds/MacOSX directory. If you don't already have CommandLineTools, I think you can get it by running `xcode-select --install` or by installing Homebrew.

**Comment by kaosbeat** [comment]
Hey, 

it appears you need to download/install the full package. I have the command-line tools installed.

```
FirmwareSender $ xcodebuild -configuration Release
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
```
